docs/troubleshoot/upgrade-zowe-no-downtime.md

```diff
@@ -0,0 +1,118 @@
+
+## Limitations
+
+There is a known limitation with balancing traffic between different versions of the UI components, in particular about the API Catalog.
+Only one API Catalog instance can be active in an installation with multiple maintainance levels.
+
+## Pre-requisites before installation upgrade with HA
+
+When upgrading an installation in high availability mode, one or more instances run at different maintainance levels. As such, a few restrictions apply:
+
+- HA settings are the same across both installations.
+- Each instance has its own configuration file (e.g. `zowe.yaml` file), and the HA and security settings are synchronized.
+- Onboarded services must be also be in high availability mode.
+
+## Upgrading installation with HA
+
+Upgrading installation to use high availability mode involves two general steps:
+
+1. Install a new Zowe instance with the upgraded version.
+2. Switch traffic to the new instance to avoid downtime.
+
+### Installing a new Zowe instance
+
+Use the following procedure to install a new Zowe instance:
+
+1. Use one of the supported installation methods to perform a separate installation of the new version of Zowe. For more information about supported installation methods, see the [Installation roadmap](../user-guide/install-zos.md).
+
+    **Note:** When performing a separate installation of Zowe, ensure that datasets, STCs definitions, and the USS installation directory are distinct from the original installation.
+
+2. Replicate the configuration of instance A into instance B.
+
+    The settings across both instances must be the same, except for properties pointing to datasets and z/OS USS filesystems which are now separated.
+
+    The following is a list of properties that will be different between the instances:
+
+    ```yaml
+    zowe.setup.datasets
+    zowe.setup.security
+    zowe.runtimeDirectory
+    zowe.logDirectory
+    zowe.workspaceDirectory
+    zowe.job
+    ```
+
+3. Choose which instance you intend to run the UI services (API Catalog, Zowe Desktop), and disable these mentioned components in the other instance:
+
+    ```yaml
+    api-catalog:
+        enabled: false
+
+    app-server:
+        enabled: false
+    ```
+
+### Switching traffic
+
+Switching traffic without incurring downtime involves the following general steps. Details of these steps are described later in this section.
+
+1. Disable an application instance (`QUIESCE`) from the DVIPA of the sysplex distributor.
+2. Stop this original instance.
+3. Start the new instance.
+4. Verifying service replication.
+5. Resume traffic to the instance.
+
+**Note:** Only new connections can be routed to the right instance. <!--What does "right" instance mean? The newly created instance? -->A decision needs to be made on long-lived connections, such as long running requests and Web Socket sessions. These connections cannot be re-routed. As such, these connections are closed when the instance is stopped.
```

Comment:
What is a "right" instance in this context?

Reply:
I updated the wording, hope it's clearer